_episodes/09-tdd.md

```diff
@@ -1,13 +1,18 @@
 ---
-layout: page
-title: Testing
-subtitle: Test Driven Development
-minutes: 10
+title: Test Driven Development
+teaching: 10
+exercises: 0
+questions:
+- "How do you make testing part of the code writing process?"
+objectives:
+- "Learn about the benefits and drawbacks of Test Driven Development."
+- "Write a test before writing the code."
+keypoints:
+- "Test driven development is a common software development technique"
+- "By writing the tests first, the function requirements are very explicit"
+- "TDD is not for everyone"
+- "TDD requires vigilance for success (cheating leads to failure)"
```

Comment:
I don't like the paranthetical "(cheating leads to failure)". That's not the point.